src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveService.kt

```diff
@@ -150,14 +166,14 @@ class OppgaveService constructor(
         return oppgaveRestClient.opprettOppgave(oppgave)
     }
 
-    fun ferdigstill(oppgaveId: Long) {
+    fun ferdigstill(oppgaveId: Long, versjon: Int?) {
         val oppgave = oppgaveRestClient.finnOppgaveMedId(oppgaveId)
 
         when (oppgave.status) {
             StatusEnum.OPPRETTET, StatusEnum.AAPNET, StatusEnum.UNDER_BEHANDLING -> {
                 val patchOppgaveDto = oppgave.copy(
                     id = oppgave.id,
-                    versjon = oppgave.versjon,
+                    versjon = versjon ?: oppgave.versjon,
```

Comment:
Jeg tenker nesten vi skulle kunna feile her i integrasjoner hvis man ikke har riktig versjon, så trenger vi ikke å kalle oppgave :) Samtidig som det er oppgave som eier denne logikken

Reply:
Vi må da uansett sende et kall til oppgave, fordi den må hentes først. Vet ikke hva som er best av å hente oppgave og feile hvis den ikke stemmer eller få tilbake feil ved forsøk på oppdatering.

Reply:
Løst ved å lage en egen funksjon for å validere versjon og kaste 409 om nødvendig